src/app/index.ts

```diff
@@ -288,20 +224,20 @@ module.exports = yo.extend({
         /** Copy the manifest */
         this.fs.copyTpl(this.templatePath(`manifest/${this.project.hostInternalName}.xml`), this.destinationPath(`${this.project.projectInternalName}-manifest.xml`), templateFills);
 
-        if (this.project.framework === 'manifest-only') {
-          this.fs.copyTpl(this.templatePath(`manifest-only/**`), this.destinationPath(), templateFills);
+        if (this.project.projectType === manifestOnly) {
+          this.fs.copyTpl(this.templatePath(manifestOnly.concat(`/**`)), this.destinationPath(), templateFills);

```

Comment:
Use a template literal here for consistency with the others: `` `${manifestOnly}/**` ``